src/hypermedia-constants.js

```diff
@@ -56,6 +56,7 @@ export const Rels = {
 		myOrganizationActivities: 'https://activities.api.brightspace.com/rels/my-organization-activities',
 		myOrganizationActivitiesEmpty: 'https://activities.api.brightspace.com/rels/my-organization-activities#empty',
 		activityCollection: 'https://activities.api.brightspace.com/rels/activity-collection',
+		instanceCollection: 'https://activities.api.brightspace.com/rels//InstanceCollection/',
```

Comment:
`//` double slash intentional? 

Moving forward we should probably change this to `instance-collection` to match everything else

Reply:
no it was not intentional, thanks for catching that